clients/rust/Cargo.toml

```
@@ -9,14 +9,18 @@ license-file = { workspace = true }
 edition = { workspace = true }
 
 [features]
+fetch = ["dep:solana-client", "dep:solana-sdk"]
+serde = ["dep:bincode", "dep:serde"]
```

We can have this enable kaigan's serde feature and disable it by default in line 19
```suggestion
serde = ["dep:bincode", "dep:serde", "kaigan/serde"]
```